internal/checker/interpreter.go

```diff
@@ -35,13 +38,34 @@ func ExecuteLuaMain(script string, importLoopData *shared.ImportLoopData, ruleCo
 }
 
 func attachRuleRequiring(importLoopData *shared.ImportLoopData, ruleConf *RuleConf, L *lua.LState) {
-	L.SetGlobal("require_rule", L.NewFunction(func(state *lua.LState) int {
-		ruleUrl := L.Get(1).String()
+	L.SetGlobal("require_remote", L.NewFunction(func(state *lua.LState) int {
+		rulesetIdentifier := L.Get(1).String()
 		ruleName := L.Get(2).String()
 
-		result := _internalCheckRule(importLoopData, ruleUrl, ruleName, ruleConf)
+		result := _internalCheckRule(importLoopData, rulesetIdentifier, ruleName, ruleConf)
 		L.Push(lua.LBool(result))
 
 		return 1
 	}))
+
+	L.SetGlobal("require_file", L.NewFunction(func(state *lua.LState) int {
+		rulesetPath := L.Get(1).String()
+
+		scriptContents, err := os.ReadFile(rulesetPath)
+		if err != nil {
+			importLoopData.InfoApi.Error(err.Error())
+			os.Exit(1)
+		}
+
+		rulesetPath, err = filepath.Abs(rulesetPath)
+		if err != nil {
+			importLoopData.InfoApi.Error("Cannot convert file path to the absolute path!")
+			os.Exit(1)
+		}
+
+		doesRulePass, _ := CheckRuleScript(importLoopData, string(scriptContents), filepath.Dir(rulesetPath))
+		L.Push(lua.LBool(doesRulePass))
```

Comment:
I though about importing in meaning of adding functions and variables from other file to the stack.
e.g. if function `test` is declared in file `example.lua` and `rule.lua` imports `example.lua`, function `test` should be available in `rule.lua`

What do you think about it?

Reply:
OK, it makes sense